syntax = "proto3";

package drltt_proto;;

import "drltt_proto/dynamics_model/basics.proto";

message State {
    optional BicycleModelState bicycle_model = 1;
}

message BicycleModelState {
    BodyState body_state = 1;
    // velocity in [m/s]
    optional float v = 2;
}
